app/models/admin/action_log_filter.rb

```diff
@@ -94,6 +95,9 @@ def scope_for(key, value)
     when 'target_account_id'
       account = Account.find_or_initialize_by(id: value)
       Admin::ActionLog.where(target: [account, account.user].compact)
+    when 'target_domain'
+      instance = Instance.find(TagManager.instance.normalize_domain(value))
+      Admin::ActionLog.where(route_param: instance.domain, target_type: %w{DomainBlock DomainAllow UnavailableDomain})
```

Comment:
I'm not sure if we really want the filter on `target_type` here (which isn't indexed, but I've added an index to route_param)

Reply:
(I've since removed this filter, but left it in place for the migration)

Reply:
It might mean the index cannot be used, so I would not do it (or at least check the query).

Also, if some future type used the domain as `route_param`/`human_identifier`, I guess it would not be wrong to return this here.

Reply:
We don't have any index usage here, so I'm thinking it's fine (let's continue this [here](https://github.com/mastodon/mastodon/pull/27386#discussion_r1685787120) since the code has changed)